docs/commands.md

```diff
@@ -444,6 +444,14 @@ cpflow run -a $APP_NAME --entrypoint /app/alternative-entrypoint.sh -- rails db:
 cpflow setup-app -a $APP_NAME
 ```
 
+### `generate`
```

Comment:
We need to fix the script that generates this file to take into account subcommands.

Reply:
Fixed the script